documentation/book/ref-kafka-user.adoc

```diff
@@ -198,6 +206,12 @@ spec:
         operation: Read
 ----
 
+=== Super user access to Kafka brokers
```

Comment:
Actually, this is configured in the broker. I do not think it should be here because it is not configured in the KAfkaUser resource. For OCP this seems to be already covered in here: https://strimzi.io/docs/master/full.html#super_users

Reply:
But what if the reader comes here first? Isn't it useful to let them know of the impact of that configuration on ACLs?

Reply:
Ok, if you think we can leave it here as well.